tests/scripts/check-files.py

```diff
@@ -182,17 +182,17 @@ def __init__(self, log_file):
         self.setup_logger(log_file)
         self.extensions_to_check = (
             ".c",
+            ".data",
+            ".function",
             ".h",
-            ".sh",
+            ".md",
             ".pl",
             ".py",
-            ".md",
-            ".function",
-            ".data",
-            "/Makefile",
-            "/Makefile.inc",
+            ".sh",
             "/CMakeLists.txt",
             "/ChangeLog"
```

Comment:
These two lines should be swapped.

Reply:
No, the order is right. But the `ChangeLog` line was missing a comma! Fixed in rebase.

Reply:
Yeah, you're right.